src/java.base/share/classes/java/io/PrintWriter.java

```diff
@@ -132,8 +133,9 @@ public PrintWriter(OutputStream out) {
     /**
      * Creates a new PrintWriter from an existing OutputStream.  This
      * convenience constructor creates the necessary intermediate
-     * OutputStreamWriter, which will convert characters into bytes using the
-     * default charset.
+     * OutputStreamWriter, which will convert characters into bytes using
+     * the charset in {@code out} if it is a {@code PrintStream}, or using
+     * the default charset.
      *
```

Comment:
I think I prefer the wording in OutputStreamWriter because it puts the default encoding first and makes it just a bit clearer that the PS case is the exception.

Reply:
Will use `OutputStreamWriter`'s wording here. Also I am tempted to make `PrintStream::charset()` public, as some custom `OutputStreamWriter` implementations would also need the charset information.

Reply:
> Also I am tempted to make `PrintStream::charset()` public, as some custom `OutputStreamWriter` implementations would also need the charset information.

I think that would be good addition.